.github/actions/broken-links-pr.js

```diff
@@ -0,0 +1,31 @@
+const { Octokit } = require('@octokit/action')
+
+const octokit = new Octokit()
+
+const [owner, repo] = process.env.GITHUB_REPOSITORY.split('/')
+
+async function main() {
+  const prs = await octokit.pulls.list({
+    owner,
+    repo,
+    state: 'open',
+    head: `${owner}:action-remove-broken-links`,
+  })
+
+  if (prs.data.length === 0) {
+    const pr = await octokit.pulls.create({
+      owner,
+      repo,
+      title: 'chore: delete apps with broken links',
+      base: 'master',
+      head: 'action-remove-broken-links',
+      body: 'Auto-update from GitHub Actions.',
```

Comment:
Can the body be the list of apps and reason why they remove. Watch to blank page without info and give the review it's hard.